### PR TITLE
EC-14395 Make it so fields aren't required anymore for updates

### PR DESCRIPTION
**Pull request by thomasio101**
We've made it so certain fields aren't required anymore when updating entities.

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

**Comment by thomasio101** [comment]
@microsoft-github-policy-service agree [company="Elfsquad"]

**Comment by thomasio101** [comment]
@microsoft-github-policy-service agree company="Elfsquad"

File: certified-connectors/Elfsquad Data/apiDefinition.swagger.json [hunk 1]
```diff
@@ -194,6 +194,27 @@
         "x-ms-visibility": "internal"
       }
     },
+    "/data/1/{entity_name}/$patch-schema": {
+      "get": {
+        "responses": {
+          "default": {
```
Comment (Amjed-Ayoub) [inline certified-connectors/Elfsquad Data/apiDefinition.swagger.json:200]: @thomasio101 default responses are not allowed. please define a proper [response](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).

File: certified-connectors/Elfsquad Data/apiDefinition.swagger.json [hunk 2]
```diff
@@ -194,6 +194,27 @@
...
+            "description": "default",
+            "schema": {}
```
Comment (Amjed-Ayoub) [inline certified-connectors/Elfsquad Data/apiDefinition.swagger.json:202]: @thomasio101 empty schema is not allowed. Please define a proper schema.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: certified-connectors/Elfsquad Data/apiDefinition.swagger.json (hunk 2 above)
Comment (thomasio101) [reply certified-connectors/Elfsquad Data/apiDefinition.swagger.json]: @Amjed-Ayoub, I've resolved this for you :)

File: certified-connectors/Elfsquad Data/apiDefinition.swagger.json (hunk 1 above)
Comment (thomasio101) [reply certified-connectors/Elfsquad Data/apiDefinition.swagger.json]: @Amjed-Ayoub, duly noted!

**Comment by thomasio101** [comment]
@Amjed-Ayoub, GitHub is saying I haven't addressed the requested changes but I believe I have. Please tell me whether these changes are what you wanted to see.

**Review by Amjed-Ayoub — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @thomasio101,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**